sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/util/BinaryData.java

```diff
@@ -185,16 +187,44 @@ public static BinaryData fromBytes(byte[] data) {
         return new BinaryData(data);
     }
 
+    /**
+     * Serialize the given {@link Object} into {@link BinaryData} using json serializer which is available in classpath.
+     * The serializer must implement {@link JsonSerializer} interface. A singleton instance of {@link JsonSerializer}
```

Comment:
I think this sentence can be left out. Either an implementation will be in the classpath or an exception will be thrown on how to resolve the issue.